src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -85,32 +85,48 @@ private boolean validateCreationDate(LocalDate creationDate, Errors errors, Stri
                 && DateValidators.isDateInPast(creationDate, qualifiedFieldName, errors, loggingContext);
     }
 
-    private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
-                                   TrustDataDto trustDataDto,
-                                   Errors errors,
-                                   String loggingContext) {
+    private void checkCeasedDateAgainstBeneficialOwners(OverseasEntitySubmissionDto overseasEntitySubmissionDto,
+                                                        TrustDataDto trustDataDto,
+                                                        Errors errors,
+                                                        String loggingContext) {
         final String qualifiedFieldName = getQualifiedFieldName(OverseasEntitySubmissionDto.TRUST_DATA, TrustDataDto.CEASED_DATE_FIELD);
+        final LocalDate creationDate = trustDataDto.getCreationDate();
         final LocalDate trustCeasedDate = trustDataDto.getCeasedDate();
 
-        if (isTrustStillRequired(overseasEntitySubmissionDto, trustDataDto.getTrustId())) {
-            // Cease date of the trust must be 'null' as there are still Individual and/or Corporate BOs associated with this trust
-            if (Objects.nonNull(trustCeasedDate)) {
-                final String errorMessage = ValidationMessages.NULL_ERROR_MESSAGE.replace("%s", qualifiedFieldName);
-                setErrorMsgToLocation(errors, qualifiedFieldName, errorMessage);
-                ApiLogger.infoContext(loggingContext, errorMessage);
+        if (areBeneficialOwnersStillLinkedToTrusts(overseasEntitySubmissionDto, trustDataDto.getTrustId())) {
+
+            if (isTrustInvolvedInOverseasEntityFlagNull(trustDataDto.isTrustInvolvedInOverseasEntity(), errors, loggingContext)) {
```

Comment:
for the  isTrustInvolvedInOverseasEntityFlagNull method, i would name it ...NotNull, otherwise it reads like we go inside that if statement only if the flag is null

Reply:
changed this.